### PR TITLE
Added Id field to /admin page, changed smiley to unicode

### PR DESCRIPTION
**Pull request by Goro789**
Added Id field to the table on the admin page.
Smiley is now unicode.
The table on the All Checkins page can be ordered on all fields.

### CODE REVIEW

**Comment by ometa** [comment]
This is exactly the sort of thing I'm looking for!  The next challenge for you is to add the accompanying test coverage so we have surety that the things we're changing render as expected.

**Comment by Goro789** [comment]
Tables in /admin can now be sorted on all fields. 
@ometa Is it ok to allow sorting on all fields or would you prefer to sort only on Id, Name etc?

File: templates/_scoreboard.erb [hunk 1]
```diff
@@ -22,7 +23,12 @@
...
         </tr>
       <% end %>
-
     </table>
   <% end %>
 </div>
+
+<script>
+    $(document).ready(function() {
+        $('.table').DataTable({bFilter: false, bInfo: false, bPaginate: false});
```
Comment (Goro789) [inline templates/_scoreboard.erb]: @ometa The values here can be changed to enable Searching, Pagination and to display some information about the number of rows. I have set all to false for now as I think pagination and info are unnecessary but bFilter brings in a search box that could be useful. This also applies to the table the users see in the all_checkins.erb file.

**Comment by Goro789** [comment]
@ometa I'm taking a look at the tests now. It may take a little while to figure out whats going on but I will try to get them done in the next day or so.

**Comment by ometa** [comment]
Manual local tests all pass, looks great.  Merging in the interest of getting this year's documentation updated and printed; we can knock out the tests in a future PR.

🚀

**Pull request merged**